### ISSUE TITLE
Optimization Method

### ISSUE THREAD

**Issue by ankitesh97** [ZichaoLong/PDE-Net#9]
I was wondering why did you use BFGS optimization instead of inbuilt ADAM/Gradient descent 
optimization method in pytorch?

**Comment by ZichaoLong** [comment]
BFGS with line search converges quiker.

**Comment by ZichaoLong** [comment]
> I was wondering why did you use BFGS optimization instead of inbuilt ADAM/Gradient descent
> optimization method in pytorch?

BFGS with line search converges quicker. ADAM and SGD with a fixed learning rate  are not stable.
